src/Components/ActivationKeys/ActivationKeys.js

```diff
@@ -43,6 +43,27 @@ const ActivationKeys = () => {
     setisOpen(!isOpen);
   };
 
+  const popoverContent = (
+    <TextContent className="pf-u-font-size-sm">
+      <Text>
+        Activation keys assist you in registering systems. Metadata such as
+        role, system purpose, and usage can be automatically attached to systems
+        via an activation key, and monitored with &nbsp;
+        <a
```

Comment:
Should this use a `Link` component to navigate within react router? That way it doesn't trigger a full page reload

Reply:
You are right, but in this case it's an external link and it supposed to open in a new tab